plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1799,6 +1799,8 @@ elementgui.enchantment.damage_modifier=<html>Damage reduction modifier:<br><smal
 elementgui.enchantment.treasure_enchantment=Is this enchantment treasure enchantment?
 elementgui.enchantment.curse=Is this enchantment curse?
 elementgui.enchantment.allowed_on_books=Is this enchantment allowed on books?
+elementgui.enchantment.can_generate_in_loots=Can this enchantment generate in loots?
```

Comment:
Change to "Can this enchantment appear in loot tables?" or something like this.  I would also adapt variable name. Adapt help tip too

Reply:
Done!